### DISCUSSION TITLE
The video is unplayable after downloading

### DISCUSSION THREAD

**Discussion by jankronlundtampere** [aclap-dev/video-downloadhelper#331] [Help and Q&A]
### Your question

Hello,
I purchased the paid version of video-downloadhelper for chrome.
It has been working well for a few weeks for downloading paid courses on Udemy.

But today I wanted to download another video of the course but it shows me a message that I need to download and install a Companion App.
So I followed the link and download and install.

But until now, the video downloaded with video-downloadhelper, when playing, shows colors randomly, green, blue, gray, ... and is without audio.

Please help me.

**Video DownloadHelper
Version 8.2.2.9 (stable)
Browser language: fr
Generated for production
Generated on 2024-03-19
Build options: browser=chrome, linuxlic=false, noyt=true, target=google

**Companion app installed: VdhCoApp 2.0.19

**Platform Win x86-64
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/122.0.0.0 Safari/537.36

### Addon & CoApp version and details

**Video DownloadHelper
Version 8.2.2.9 (stable)
Browser language: fr
Generated for production
Generated on 2024-03-19
Build options: browser=chrome, linuxlic=false, noyt=true, target=google

**Companion app installed: VdhCoApp 2.0.19

**Platform Win x86-64
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/122.0.0.0 Safari/537.36

**Comment by Mattmjs** [comment]
DRM protection, no way around that. See a another discussion that confirmed it https://github.com/aclap-dev/video-downloadhelper/discussions/49#discussioncomment-8425798
